src/sass/modules/_m-button.scss

```diff
@@ -178,7 +178,7 @@ button.short {
 }
 
 .va-btn-close-icon {
-  background: transparent url(assets/img/icons/close.svg) center no-repeat;
+  background: transparent url(/img/icons/close.svg) center no-repeat;
```

Comment:
`resolve-url-loader` was fixing this path before? And this is the only case of that `asset` prefix? We were making a big sacrifice on the build time just for that lol.

Reply:
It was mostly for the paths in the uswds sass code. It looks like this is the only place where our code needed it, though.